backtester/eventhandlers/strategies/strategies.go

```diff
@@ -15,31 +16,31 @@ import (
 
 // LoadStrategyByName returns the strategy by its name
 func LoadStrategyByName(name string, useSimultaneousProcessing bool) (Handler, error) {
-	strategies := GetStrategies()
+	strategies := GetSupportedStrategies()
 	for i := range strategies {
 		if !strings.EqualFold(name, strategies[i].Name()) {
 			continue
 		}
-		if useSimultaneousProcessing {
-			if !strategies[i].SupportsSimultaneousProcessing() {
-				return nil, fmt.Errorf(
-					"strategy '%v' %w",
-					name,
-					base.ErrSimultaneousProcessingNotSupported)
-			}
-			strategies[i].SetSimultaneousProcessing(useSimultaneousProcessing)
+		// create new instance so strategy is not shared across all tasks
+		s, ok := reflect.New(reflect.ValueOf(strategies[i]).Elem().Type()).Interface().(Handler)
```

Comment:
🤣 One way to do it. Suggestion: Elem and Interface calls panic if underlying is nil, so maybe breaking up the calls and doing some nil checks might be betterer.

Reply:
Underlying shouldn't be `nil` as `supportedStrategies` is a local var that doesn't get modified and is filled with `new` instances of strategies. Plugin based strategies also have a `nil` check before being added to the `supportedStrategy` var.

Though I don't like panics, so I'll add protections anyway